mypyc/primitives/misc_ops.py

```diff
@@ -55,21 +55,19 @@
     error_kind=ERR_NEVER)
 
 # This is sort of unfortunate but oh well: yield_from_except performs most of the
-# error handling logic in `yield from` operations. It returns a bool and a value.
+# error handling logic in `yield from` operations. It returns a bool and passes
+# a value by address.
 # If the bool is true, then a StopIteration was received and we should return.
 # If the bool is false, then the value should be yielded.
 # The normal case is probably that it signals an exception, which gets
 # propagated.
-yield_from_rtuple = RTuple([bool_rprimitive, object_rprimitive])
-
 # Op used for "yield from" error handling.
 # See comment in CPy_YieldFromErrorHandle for more information.
-yield_from_except_op = custom_op(
-    name='yield_from_except',
-    arg_types=[object_rprimitive],
-    result_type=yield_from_rtuple,
-    error_kind=ERR_MAGIC,
-    emit=simple_emit('{dest}.f0 = CPy_YieldFromErrorHandle({args[0]}, &{dest}.f1);'))
+yield_from_except_op = c_custom_op(
+    arg_types=[object_rprimitive, object_rprimitive],
```

Comment:
Here the `object_rprimitive` is potentially confusing. Add comment explaining the real type of the second argument. Alternatively, if we'd use `pointer_rprimitive`, this would be arguably be cleaner. 

Another random idea: what about adding a new primitive type corresponding to `PyObject **`? We could call it `object_pointer_rprimitive`, perhaps.